src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -197,6 +197,7 @@ public function onKernelResponse(ResponseEvent $event)
         if ($autoCacheControl) {
             $response
                 ->setExpires(new \DateTimeImmutable())
+                ->setNoStore()
```

Comment:
I personally would not do this by default. I mean its fine in most cases even when having a session that History Back of the browser does not make any request to the server and just load the content from the browser history and don't use server resources.

Still think the method make sense and can for this who need it be called in the `index.php` to make all there responses noStore.